crates/levm/src/vm.rs

```diff
@@ -9,6 +9,15 @@ pub struct VM {
     pub pc: usize,
 }
 
+/// Shifts the value to the right by 255 bits and checks the most significant bit is a 1
+fn is_negative(value: U256) -> bool {
+    value >> 255 == U256::one()
+}
+/// Converts a positive value to a negative one in two's complement
+fn to_negative(value: U256) -> U256 {
```

Comment:
Can we rename this to `negate` or similar? Since it also turns negative numbers into positive ones.